snuba/web/rpc/__init__.py

```diff
@@ -101,5 +112,44 @@ def _after_execute(
 }
 
 
-for version, module_path in _TO_IMPORT.items():
-    import_submodules_in_directory(module_path, f"snuba.web.rpc.{version}")
+for v, module_path in _TO_IMPORT.items():
+    import_submodules_in_directory(module_path, f"snuba.web.rpc.{v}")
+
+
+def run_rpc_handler(name: str, version: str, data: bytes) -> Response:
```

Comment:
write a test for this please

Reply:
i added a test for the exception being returned - was there something else you had in mind?

Reply:
the actual happy path is already tested by the _simple tests

Reply:
What's untested:

1. failed deserialization
2. the endpoint throwing an exception on execution and this function returning an ErrorProto

I know there are endpoint tests but this will also be used by snuba-admin and I would like this to be tested as its own unit, without the flask app as an intermediary